src/components/homepage/Homepage.jsx

```diff
@@ -1,8 +1,17 @@
+import { useState, useEffect } from 'react'
+import Movie from '../movie/Movie'
+import useFetch from '../../hooks/useFetch';
+import { MovieContainer } from './Styles/StyleMovieContainer';
+
 const Homepage = () => {
+    const url = 'http://localhost:8001/movies';
```

Comment:
make env file and add this `http://localhost:8001` there. After that create an API folder with movies.js file: `src/API/movies.js` Create a function called getMovies() and fetch the data. Use the env variable there and the rest of the url. Import that function here and use it.

Reply:
Done